### DISCUSSION TITLE
feature that shows that the user is speaking by highlighting a circle around them.

### DISCUSSION THREAD

**Discussion by kashyapsanchit** [benawad/dogehouse#2542] [Q&A] [answered]
I am really curious to know about the library or the functionality one needs to implement when a user is speaking ( it shows a green outline on the user that is speaking ). Will be a really great favor if you can share.

**Comment by amitojsingh366** [comment, marked as answer]
We just send a ws event when the speaker's volume goes above a certain level, then that ws event is handled client side and we show the circle

**Reply by kashyapsanchit** [reply]
Thanks man !
